appdata.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.geogebra.GeoGebra.desktop</id>
+  <name>GeoGebra</name>
+  <developer_name>GeoGebra</developer_name>
+  <summary>Dynamic mathematics software</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
Reading more in their license page...
- "You are free to copy, distribute and transmit GeoGebra for non-commercial purposes".
- The source code is "GPL v3 or later" and can be found here https://github.com/geogebra/geogebra
- The language files for translations and artwork like the icon I added here are: Creative Commons Attribution-NonCommercial-ShareAlike license (version 3.0 or later)